source/MaterialXTest/ShaderValid.cpp

```diff
@@ -909,23 +1026,34 @@ TEST_CASE("MaterialX documents", "[shadervalid]")
                 }
                 if (nodeDef)
                 {
+                    setupTime.endTimer();
```

Comment:
End setup timer when we go to measure the per language execution then start up again afterwards.